README.md

```diff
@@ -1,14 +1,94 @@
-### shared-latent-space
-Shared Latent Space VAE's
+# Shared Latent Space Variational Autoencoders
 
-## main_file.py
-This is the file which should be called. It handles loading data and formatting it. It also calls upon shared_vae_class.py to create the mode, train it, and generate data. As work continues, this file will become more general and easier to work with.
+## Motivation for application of Shared Latent SPace VAE's In Biology
 
-## shared_vae_class.py
-This file is the main class which hold the model. It contains functions to compile, train, and generate from the model. The model will take in a series of parameters which control size of layers, etc. The model right now is very rigid in structure, but this may change.
+Variational Autoencoders are machine learning models which learn the distribution of the data in a latent space manifold.
+This allows the model to be trainned on unsuprivesed data, but learning to recreate the input data, and also allows the model to
+create new data that ressembles the original data by picking points in the lantent space manifold.
+Shared Latent Space VAE's find relationships between two different domains and allow for transformations between the two.
+They achieve this by linking the lantent space manifold between two differen encoders and decoders.
+This is particularly useful in Biology where we could use different data types as different 'views'
+on the same biological problem.
+The ability to transform between domains also allows us to transition between different data types.
+
+## Diagram of Model
+![Alt text](Shared_Latent_Space_VAE.png)
+
+## Usage
+
+### Computational Environment
+
+All libraries and packages are handled by conda and specified in the `enviroment.yml` file.
+To build and activate this enviroment, run:
+```
+#conda version 4.4.10
+conda env create --force --file enviroment.yml
```

Comment:
Is there a reason to have force here?

Reply:
The force is so that it will overwrite an existing environment with the same name. It isn't explicitly needed, but if the environment changes, it'll make sure that it updates.